wled00/FX.h

```diff
@@ -44,8 +44,20 @@
 
 /* Not used in all effects yet */
 #define WLED_FPS         42
-#define FRAMETIME_FIXED  (1000/WLED_FPS)
 #define FRAMETIME        strip.getFrameTime()
+#if defined(ARDUINO_ARCH_ESP32) && !defined(CONFIG_IDF_TARGET_ESP32C3)    // all ESP32 except -C3(very slow, untested)
+  #define FRAMETIME_FIXED  (strip.getFrameTime() < 10 ? 12 : 24)            // allow faster FRAMETIME_FIXED when target FPS >= 100  
+  #define MIN_SHOW_DELAY   (max(2, (_frametime*5)/8))                       // supports higher framerates and better animation control -- 5/8 = 62%
+  // used to initialize for strip attributes:
+  #define WLED_FPS_SLOW    42
+  #define FRAMETIME_FIXED_SLOW  (24)                                        // 1000/42 = 24ms
+#else
+  #define FRAMETIME_FIXED  (1000/WLED_FPS)
+  #define MIN_SHOW_DELAY   (_frametime < 16 ? 8 : 15)                       // legacy MIN_SHOW_DELAY - creates more idle loops, but reduces framerates
+  #define WLED_FPS_SLOW     WLED_FPS
+  #define FRAMETIME_FIXED_SLOW FRAMETIME_FIXED
+#endif
+#define FPS_UNLIMITED    119
```

Comment:
I'd recommend using '0' for the "unlimited" mode.  0 happens to be a magic number that can be tested for with one fewer instruction everywhere we need to, and is an otherwise "meaningless" value for an FPS limit.

Reply:
Good point - actually it would need a small change in the UI, but I like the idea of saying "0 fps = unlimited".

Reply:
I'll double-check if `getTargetFPS() == 0` can cause problems like div/0 anywhere in the code.